setup/www/ansible-playbook.yaml

```diff
@@ -24,8 +24,12 @@
       with_items: packages
       tags: general
 
-    - name: User | Add {{ server_user }} user
-      user: name="{{ server_user }}" shell=/bin/bash
+    - name: User | Add system users
+      user: name="{{ item }}" shell=/bin/bash
+      with_items:
+        - "{{ server_user }}"
+        - "{{ staging_user }}"
+        - "{{ dist_user }}"
```

Comment:
@rvagg we're now creating all three of the utility accounts we use, you can see that here. I have a couple of questions lower in the playbook.